models.py

```diff
@@ -905,7 +905,7 @@ def _format_totalclaimed_column_export(view, context, model, name):
     form_args = dict(
             reason=dict(label='Reason', validators=[DataRequired()])
             )
-    form_extra_fields = {'recipient': TextField('Recipient', validators=[DataRequired()]), 'message': TextField('Message'), 'amount': DecimalField('Amount', validators=[validators.Optional(), DataRequired()]), 'csvfile': FileField('CSV File')}
+    form_extra_fields = {'recipient': TextField('Recipient'), 'message': TextField('Message'), 'amount': DecimalField('Amount', validators=[validators.Optional()]), 'csvfile': FileField('CSV File')}
```

Comment:
Datarequired forces you to input something in those fields. This breaks CSV file option because you will not be able to upload csv file until you fill in the fields.